### PR TITLE
Added NaiveBayes Model description.

### PR DESCRIPTION
**Pull request by safzalm**

### CODE REVIEW

File: source/main.Rmd [hunk 1]
```diff
@@ -431,6 +431,13 @@ rm(trainDataCopy,testDataCopy,ctrl)
...
+
+It is simple (both intuitively and computationally), fast, performs well with small amounts of training data, and scales well to large data sets. The greatest weakness of the naïve Bayes classifier is that it relies on an often-faulty assumption of equally important and independent features which results in biased posterior probabilities. Although this assumption is rarely met, in practice, this algorithm works surprisingly well and accurate; however, on average it rarely can compete with the accuracy of advanced tree-based methods (random forests & gradient boosting machines) but is definitely worth having in our toolkit.
+
+The Baive Bayes algorithm can be implemented using different packages (i.e. e1071, klaR, naivebayes) Here we are using the caret package.The caret package contains train() function which is helpful in setting up a grid of tuning parameters for a number of classification and regression routines, fits each model and calculates a resampling based performance measure.caret allows us to use the different naïve Bayes packages but in a common framework, and also allows for easy cross validation and tuning.
```
Comment (v2msLabs) [inline source/main.Rmd]: I would suggest to skip this paragraph. Actually caret employs klaR implementation. The rest is excellent

**Review by v2msLabs — CHANGES_REQUESTED** [review]
Please review my comment

**Comment by safzalm** [comment]
NB Model description changed.

**Comment by safzalm** [comment]
Please review the changes I made in NB Model description.

**Comment by safzalm** [comment]
I am closing this and creating new PR

**Pull request closed without merging**